src/Tests/Microsoft.DotNet.ApiCompatibility.Tests/SuppressionTests/SuppressionEngineTests.cs

```diff
@@ -61,13 +70,12 @@ public void SuppressionEngineSuppressionsRoundTrip()
             (stream) =>
             {
                 stream.Position = 0;
-                using StreamReader reader = new StreamReader(stream);
+                using StreamReader reader = new(stream);
                 output = reader.ReadToEnd();
             });
             engine.WriteSuppressionsToFile("DummyFile");
```

Comment:
do we overwrite the file after each call ?

Reply:
Actually we should generate a tmp file everytime as this could lead to race conditions.

Reply:
are we planning to do it in this pr or in the future ?